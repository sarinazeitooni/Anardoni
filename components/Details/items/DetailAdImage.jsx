import React from 'react';
import style from './scss/DetailAdImage.module.scss'
function DetailAdImage(){
    return(
        <div className={style['ad-container']}>
            <img className={style.ad} src='https://anardoni.com/img/ads/anargift.gif'/>
        </div>
    )
};
export default DetailAdImage