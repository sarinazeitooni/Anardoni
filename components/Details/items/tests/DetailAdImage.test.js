import React from 'react';
import { shallow } from 'enzyme';
import DetailAdImage from "../DetailAdImage";
describe(" detail ad image component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<DetailAdImage/>)
        expect(container).toMatchSnapshot();
    });
    it("it should render an image",()=>{
        const container = shallow(<DetailAdImage/>)
        const image = container.find('img')
        expect(image).toHaveAttribute('src' , 'https://anardoni.com/img/ads/anargift.gif')
    })
});