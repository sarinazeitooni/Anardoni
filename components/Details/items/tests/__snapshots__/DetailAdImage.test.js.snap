// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` detail ad image component it should render a div 1`] = `ShallowWrapper {}`;
